setup.py

```diff
@@ -38,7 +38,7 @@ def run(self):
 
 setup_args = dict(
     name                = 'oauthenticator',
-    packages            = ['oauthenticator'],
+    packages            = ['oauthenticator', 'oauthenticator.tests'],
```

Comment:
Can we use `get_packages()` instead here?

Reply:
I've switched to `find_packages()`